Lib/ensurepip/__init__.py

```diff
@@ -27,52 +17,36 @@
 _WHEEL_PKG_DIR = sysconfig.get_config_var('WHEEL_PKG_DIR')
 
 
-def _find_packages(path):
-    packages = {}
+def _find_wheel_pkg_dir_pip():
```

Comment:
@AA-Turner I don't understand why you named it like this — it doesn't return a directory as far as I see. It looks like it returns a replacement package...

Reply:
The function locates pip in the wheel package directory. I don't see it as "replacement" as the point of the ``--with-wheel-pkg-dir`` option is that distributors remove the bundled wheels in ``_bundled``, so there is no 'original' wheel to replace from that distributor's perspective.

Reply:
Well, until the downloader PR is merged, there's the original "source" wheel right in the Git repo, and when distros want to replace it, they add an additional source wheel, making it two (unless they also delete the first one, of course). Hence, replacement...

Reply:
I don't have better suggestions so I'll just mark this as resolved.